packages/agreement-process/test/upgradeAgreement.test.ts

```diff
@@ -554,335 +545,6 @@ describe("upgrade Agreement", () => {
     }
   });
 
-  it("should succeed with valid Verified, Certified, and Declared attributes when consumer and producer are different and requester is a delegate", async () => {
```

Comment:
Unrelated, removing this test because the upgrade does not involve producer delegation, the test passed because it called the function authenticating as the consumer, despite the test name (see the line where authData is created).